corehq/ex-submodules/soil/util.py

```diff
@@ -41,17 +41,18 @@ def expose_cached_download(payload, expiry, file_extension, mimetype=None,
     return ref
 
 
-def expose_file_download(path, **kwargs):
+def expose_file_download(path, expiry, **kwargs):
```

Comment:
What do you think of changing the units for `expiry` (for `expose_file_download` and `expose_blob_download`) to minutes instead of seconds? I doubt anyone will ever want to have a file download available for less than one minute, and then it would be consistent with the blobdb timeout parameter units.

Reply:
Could do, would just need to be converted to seconds before passing to cache. I'm going to leave it for now.

When making this change I actually considered changing the blobdb expiry to seconds since that's more consistent with everything else and more in line with what developers expect (I think)